libs/utils/src/filenames.ts

```diff
@@ -118,6 +120,30 @@ export function generateCastVoteRecordReportDirectoryName(
   return isTestMode ? `TEST${SECTION_SEPARATOR}${filename}` : filename;
 }
 
+/**
+ * Generates a name for a cast vote record export directory
+ */
+export function generateCastVoteRecordExportDirectoryName({
+  inTestMode,
+  machineId,
+  time = new Date(),
+}: {
+  inTestMode: boolean;
+  machineId: string;
+  time?: Date;
+}): string {
```

Comment:
Main difference between this function and the deprecated one is that this one doesn't include ballot count in the generated name. Including a ballot count in the directory name doesn't make much sense if we're continuously exporting, as the count's always changing.

I know this intermediate state with two similarly named functions is a bit confusing, but I'm going to prune a lot of old code once this project is done (in just a couple of weeks).